src/dataregistry/DREGS.py

```diff
@@ -46,22 +46,21 @@ def __init__(
 
         # If no schema specified, go to the default one
         if schema_version is None:
-            self.schema_version = SCHEMA_VERSION
-        else:
-            self.schema_version = schema_version
+            schema_version = SCHEMA_VERSION
```

Comment:
Can just pass `schema_version` from the `__init__` straight through to `DbConnection`, as it's init also defaults it to `SCHEMA_VERSION` if None?

i.e., dont need these lines anymore

        # If no schema specified, go to the default one
        if schema_version is None:
            schema_version = SCHEMA_VERSION